documentation.html

```diff
@@ -251,6 +251,7 @@ <h2>Primary Buttons</h2>
                                         class="fa fa-cloud-upload"></i></span></a>
                             <a class="primary-button icontext-button"><span class="btn-icon"><i
                                         class="fa fa-sign-in"></i></span>Sign-in</a>
+                            <a class="primary-button floating-button"><i class="fa fa-plus my-float"></i></a>
```

Comment:
why are you using anchor tag i think you need an button.

Reply:
ooh okay